scripts/delete.rb

```diff
@@ -0,0 +1 @@
+Person.where('title = :title', title:'Wormchow').destroy
```

Comment:
`.where` returns an array of results, whether or not there is more than one match. If you want this to work, you either need to write `.destroy_all` to destroy all of the results, or use `find_by` instead of `.where`